tests/integration/test_storage_nats/test.py

```diff
@@ -46,6 +47,23 @@ def wait_nats_to_start(nats_port, ssl_ctx=None, timeout=180):
         except Exception as ex:
             logging.debug("Can't connect to NATS " + str(ex))
             time.sleep(0.5)
+    
+    assert False, "NATS is unavailable"
+
+def wait_nats_paused(nats_port, ssl_ctx=None, timeout=180):
+    start = time.time()
+    while time.time() - start < timeout:
+        try:
+            asyncio.run(check_nats_is_available(nats_port, ssl_ctx=ssl_ctx))
+            time.sleep(0.5)
+        except nats.errors.NoServersError as ex:
+            logging.debug("NATS is unavailable: connect failed. error:" + str(ex))
+            return
+        except Exception as ex:
+            logging.debug("Can't connect to NATS " + str(ex))
```

Comment:
...and this logging is bad also - if we do not expect anything else but `NoServersError` - this should be a warning with a proper message something like "unexpected"...